client/src/components/modal-forms/ReturnOfServiceForm.js

```diff
@@ -126,6 +169,25 @@ export const ReturnOfServiceForm = ({
                 ]}
                 boldLabel={true}
               />
+              {values.sameSite === false && (
+                <Box mt={3} mb={2}>
+                  <Field
+                    name='site'
+                    component={RenderSelectField}
+                    label='Select Site'
+                    options={mapToOptions(sites)}
+                    onChange={(event) => handleSiteSelection(event, setFieldValue)}
+                  />
+                  <br />
```

Comment:
Instead of adding `<br />`; I'd rather wrapped these Fields within its own `Box` components - that way it's easier to manage spacing between them